<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Projekt3Class</class>
 <widget class="QMainWindow" name="Projekt3Class">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projekt3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj obiekt</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>256</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dane obiektu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lista obiektów</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edytuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyjscie</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Projekt3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Projekt3Class</receiver>
   <slot>dodajOkno2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemPressed(QListWidgetItem*)</signal>
   <receiver>Projekt3Class</receiver>
   <slot>wyswietlInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Projekt3Class</receiver>
   <slot>usun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Projekt3Class</receiver>
   <slot>zapiszDoPliku()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Projekt3Class</receiver>
   <slot>wczytajPlik()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemPressed(QListWidgetItem*)</signal>
   <receiver>Projekt3Class</receiver>
   <slot>wyswietlObrazek()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Projekt3Class</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Projekt3Class</receiver>
   <slot>edytuj()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Projekt3Class</receiver>
   <slot>wyjscie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dodajOkno2()</slot>
  <slot>wyswietlInfo()</slot>
  <slot>dodajOkno3()</slot>
  <slot>usun()</slot>
  <slot>zapiszDoPliku()</slot>
  <slot>wczytajPlik()</slot>
  <slot>wyswietlObrazek()</slot>
  <slot>edytuj()</slot>
  <slot>wyjscie()</slot>
 </slots>
</ui>
